.circleci/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM alpine:3.7
```

our other base images are 3.12 so this should reduce downloading I think..

```suggestion
FROM alpine:3.12
```